contracts/tokenomics/staking/tests/integration.rs

```diff
@@ -251,13 +325,13 @@ fn should_not_allow_enter_if_not_enough_approve() {
     assert_eq!(
         res.unwrap(),
         BalanceResponse {
-            balance: Uint128::from(100u128)
+            balance: Uint128::from(0u128)
         }
     );
 }
 
 #[test]
-fn should_not_allow_withdraw_more_than_what_you_have() {
+fn should_not_allow_enter_if_not_enough_approve() {
```

Comment:
What kind of approve does this test handle? We don't have approve any more. What's the difference between this test and cw20receive_enter_and_leave